cli.js

```diff
@@ -137,6 +137,38 @@ function buildSwapAndRepayParams(
   );  
 }
 
+function isValidAsset(asset) {
```

Comment:
lets move this one inside execute(), and use `if (!tokens[asset]) ...`, otherwise we'll need to maintain multiple places with assets.